src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -194,14 +202,21 @@ protected function processItem($WpObject)
         /*
          * Shipping address
          */
+        $shippingHouseNumber = '';
+        if (AddressSearchEndpoint::DEFAULT_COUNTRY_ISO === $WpObject->get_shipping_country(self::CONTEXT)) {
+            $houseNumber = $WpObject->get_meta('shipping_address_house_number', true);
+            if (!empty($houseNumber)) {
+                $shippingHouseNumber = $houseNumber.' ';
+            }
+        }
         if ($WpObject->has_shipping_address()) {
             $callData['shipping_address'] = [
                 'name' => $WpObject->get_formatted_shipping_full_name(),
                 'contact_address' => [
                     'state' => $WpObject->get_shipping_state(self::CONTEXT),
                     'city' => $WpObject->get_shipping_city(self::CONTEXT),
                     'zipcode' => $WpObject->get_shipping_postcode(self::CONTEXT),
-                    'street' => trim($WpObject->get_shipping_address_1(self::CONTEXT)).' '.trim(
+                    'street' => $shippingHouseNumber.trim($WpObject->get_shipping_address_1(self::CONTEXT)).' '.trim(
```

Comment:
you should send house number in `streetnumber` field

Reply:
Properly sending `streenumber` and `flatnumber` now.
![image](https://user-images.githubusercontent.com/18332309/152178702-d03b466e-0dec-4202-b402-f1e18f00bd56.png)